test/Table_Tests/src/Common_Table_Operations/Order_By_Spec.enso

```diff
@@ -34,6 +34,13 @@ type Data
             table_builder [col1, col2, col3, col4, col5, col6, col7, col8, col9, col10] connection=connection
         [connection, mk_table]
 
+    is_unexpected_collation self _setup =
+        IO.println "ABCDE "+(Meta.get_simple_type_name self.connection)
+        if Meta.get_simple_type_name self.connection != "Postgres_Connection" then False else
```

Comment:
please try
```suggestion
        if prefix.contains "Postgre" . not then False else
```

Reply:
After discussion with @radeusgd, I am simply marking these tests pending. It will require more research to know how to test this properly.